android/app/src/main/java/com/bounswe2020group3/paperlayer/util/Constants.kt

```diff
@@ -1,5 +1,5 @@
 package com.bounswe2020group3.paperlayer.util
 
-//const val BASE_URL = "https://paperlayer.herokuapp.com"
-const val BASE_URL = "https://paperlayer-backend.azurewebsites.net"
+const val BASE_URL = "https://paperlayer.herokuapp.com"
+//const val BASE_URL = "https://paperlayer-backend.azurewebsites.net"
```

Comment:
I think we shouldn't keep heroku in the master anymore. Let's just use it on local and never commit it